common/lib/xmodule/xmodule/course_module.py

```diff
@@ -423,7 +423,7 @@ class CourseFields(object):
         scope=Scope.settings, default=_('Course Handouts'))
     show_timezone = Boolean(
         help=_(
-            "True if timezones should be shown on dates in the courseware. "
+            "True if timezones should be shown on dates in the course. "
             "Deprecated in favor of due_date_display_format."
```

Comment:
this one doesn't actually appear on the page any more -- possibly becasue of being deprecated.